lib/fauna/client.rb

```diff
@@ -12,6 +12,8 @@ module Fauna
   class Client
     # The Connection in use by the Client.
     attr_reader :connection
+    # The configuration in use by the Client.
```

Comment:
Maybe not obvious enough that this is identical to the parameters passed into the constructor?